x-pack/plugin/esql/src/main/java/org/elasticsearch/xpack/esql/action/EsqlCapabilities.java

```diff
@@ -407,7 +407,7 @@ public enum Cap {
         /**
          * Supported the text categorization function "CATEGORIZE".
          */
-        CATEGORIZE_V4,
```

Comment:
Heya, now that `CATEGORIZE` is available in tech preview, it's important we do not replace the capability anymore - instead, any change that does not work on old nodes requires that we add a new capability and mark it as requirements for the corresponding tests.

Otherwise, bwc tests (esp. rolling upgrade tests for e.g. Serverless) will not be run.